recipes/grpc/all/conanfile.py

```diff
@@ -27,7 +27,8 @@ class grpcConan(ConanFile):
         "objective_c_plugin": [True, False],
         "php_plugin": [True, False],
         "python_plugin": [True, False],
-        "ruby_plugin": [True, False]
+        "ruby_plugin": [True, False],
+        "secure": [True, False]
```

Comment:
If I read this recipe correct, the build system unconditionally builds both unsecure and secure libraries.
Does it have a flag to enable/disable one of both?

If there isn't, you can do two things:
- When having `secure=True`, remove the unsecure libraries in `package` and vice versa for `secure=False`
- Or, as you're doing now, add a unsecure target when choosing `secure=False`.
  You need to remove the creation of the secure targets in this case.
  Also, it seems like the package for `secure=True` and `secure=False` is the same, so you can do:
    ````python
    def package_id(self):
        del self.info.options.secure
    ````

Reply:
@madebr I think it's almost the same case as we have in google test (and other testing/mocking libraries), where it produces `gtest` and `gtest_main` always, but you are free to link with any of them (depends if you have your own main or just use provided). AFAIK SDK2 and few other libraries use similar approaches.

Reply:
I'll take this review into account in https://github.com/conan-io/conan-center-index/pull/9044:
- remove secure in package_id()